### ISSUE TITLE
[FEATURE] Use local model as an LLM backbone

### ISSUE THREAD

**Issue by izhekov** [h2oai/h2o-llmstudio#577]
### 🚀 Feature

Make it possible to use e local model as LLM backbone.

### Motivation

Currently it is required the base model you want to train against to be hosted on huggingface. In many cases those models could be already created locally on user's machine. Having to upload and then download the model back is ineffective.

**Comment by psinger** [comment]
You can already provide a local path in the `LLM Backbone` field.

**Comment by izhekov** [comment]
Thank you. I tried again, and after several attempts, indeed it seems to work. In fact I tried it before posting this, but it seems it's not always working. There seems to be some issues with the input field as sometimes the value is cleared, while in other cases it's rejected during the training. I also didn't saw any information about it within the description of the field. Should I open a separate BUG ticket about those issues?
